src/main/java/mate/academy/spring/service/impl/AuthenticationServiceImpl.java

```diff
@@ -22,6 +24,9 @@ public User register(String email, String password) {
         User user = new User();
         user.setEmail(email);
         user.setPassword(password);
+        Role role = new Role();
+        role.setRoleName(Role.RoleName.USER);
+        user.setRoles(Set.of(role));
```

Comment:
It`s better not to create role using new, insted autowire roleService
![image](https://user-images.githubusercontent.com/124060117/236679464-e2944c19-bcb3-41a0-ae67-7005805aae79.png)